src/main/java/eu/bbmri_eric/negotiator/database/model/Network.java

```diff
@@ -0,0 +1,99 @@
+package eu.bbmri_eric.negotiator.database.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import lombok.*;
+
+@Getter
+@Setter
+@Entity
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class Network {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "resource_id_seq")
+  @SequenceGenerator(name = "resource_id_seq", initialValue = 10000, allocationSize = 1)
```

Comment:
you cannot use a generator of a different entity. you need to create a new one. Smth like: "network_id_seq".